File: app/src/main/java/toluog/quickeats/model/Table.java
package toluog.quickeats.model;

public class Table {
    private String id;
    private String restaurantId;
    private boolean occupied;
    private double total;

    public Table() {
    }

    public Table(String id, String restaurantId, boolean occupied, double total) {
        this.id = id;
        this.restaurantId = restaurantId;
        this.occupied = occupied;
        this.total = total;
    }

    public String getRestaurantId() {
        return restaurantId;
    }

    public void setRestaurantId(String restaurantId) {
        this.restaurantId = restaurantId;
    }

    public boolean isOccupied() {
        return occupied;
    }

    public void setOccupied(boolean occupied) {
        this.occupied = occupied;
    }

    public double getTotal() {
        return total;
    }

    public void setTotal(double total) {
        this.total = total;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }
}

File: app/src/main/java/toluog/quickeats/model/Table.kt
package toluog.quickeats.model

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Table(var id: String = "", var restaurantId: String = "", var isOccupied: Boolean = false,
                 var total: Double = 0.0, var occupants: ArrayList<User> = arrayListOf(),
                 var orders: ArrayList<Order> = arrayListOf()): Parcelable
